### DISCUSSION TITLE
Bash Shell - Zxce3

### DISCUSSION THREAD

**Discussion by giscus** [Zxce3/Zxce3#13] [General]
# Bash Shell - Zxce3

Bash atau (mulai sekarang) shell de facto di sebagian besar sistem yang banyak digunakan di Linux, macOS, dan WSL di Windows 10/11.

https://zxce3.net/blog/bash-shell/

**Comment by anwareset** [comment]
Mantap pak met, rapi.

**Reply by Zxce3** [reply]
terimakasih
